site/content/blogs/release-v118.en.md

```diff
@@ -0,0 +1,42 @@
+# AWS Copilot v1.18: Certificates import, ordering deployments in a pipeline and more
+
+The AWS Copilot core team is announcing the Copilot v1.18 release.
+Special thanks to [@corey-cole](https://github.com/corey-cole) who contributed for this release. Our public [сommunity сhat](https://gitter.im/aws/copilot-cli) is growing and has almost 280 people online and over 2.2k stars on [GitHub](http://github.com/aws/copilot-cli/).
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.18 brings several new features and improvements:
+
+* **Certificates import:** You can now run `copilot env init --import-cert-arns` to import validated ACM certificates to your environment for the load balancer listener. [See detailed section](./#certificates-import).
+* **Ordering deployments in a pipeline:** When use pipeline to deploy, now you are able to specify dependencies between their workload deployments, making it possible to deploy your services in order. [See detailed section](./#ordering-deployments-in-a-pipeline).
```

Comment:
```suggestion
* **Ordering deployments in a pipeline:** When using pipelines to deploy, you are now able to specify dependencies between services so that they are deployed in order. [See detailed section](./#ordering-deployments-in-a-pipeline).
```